pkg/httpserver/middleware_request_context.go

```diff
@@ -0,0 +1,23 @@
+package httpserver
+
+import (
+	"context"
+
+	"github.com/gin-gonic/gin"
+)
+
+type ContextKey string
+
+const (
+	RequestContextKey ContextKey = "http-request"
```

Comment:
Please make this a private const and a private type - there is no need to export this